src/frontend/src/common/utils/strUtils.ts

```diff
@@ -1,3 +1,9 @@
+const WORDS_TO_PLURALIZE: Record<string, string> = {
+  was: "were",
+};
+
 export const pluralize = (str: string, count: number): string => {
-  return count === 1 ? str : `${str}s`;
+  if (count === 1) return str;
```

Comment:
i'm a little confused what count is being used for here

Reply:
In English, you would make a word singular only if there is _one_ of them. For quantities less than 1 (zero), or quantities greater than 1, you would use the plural version of a word.

For example
```
0 samples  // plural
1 sample  // singular
2 samples // plural
```